vsphere/config.libsonnet

```
@@ -0,0 +1,30 @@
+{
+  // Static selector to apply to ALL dashboard variables of type query, panel queries, alerts and recording rules.
+  filteringSelector: 'job=~"integrations/vsphere"',
```

```suggestion
  filteringSelector: 'job="integrations/vsphere"',
```
I feel like this doesn't need to be a regex match since it's a hardcoded value.